services/provider/server/server.go

```diff
@@ -201,3 +216,56 @@ func newClient() (client.Client, error) {
 
 	return client, nil
 }
+
+func (s *OCSProviderServer) getPublicKey(ctx context.Context, secretName string) (string, error) {
```

Comment:
Can we please rename it to `getOnboardingValidationKey`? the current name is very generic and loses meaning without looking at the call site?